clients/drcachesim/drcachesim.dox.in

```diff
@@ -721,6 +721,14 @@ running with "-data_prefetcher none" (see \ref sec_drcachesim_ops).
 While misses from software prefetches are included in cache miss files,
 misses from hardware prefetches are not.
 
+The cache simulator can be used to analyze the stream of L2 cache miss
+addresses. This can be useful when looking for patterns that can be utilized
+in SW prefetching. The current analyzer can only identify simple stride
+patterns, but it can be extended to search for more complex patterns.
+To invoke the miss analyzer, pass \p miss_analyzer to the \p -simulator_type
+parameter.
```

Comment:
Comment that the results can be written to a file using the -LL_miss_file option.

Reply:
Done.